dev-util/gengetopt/gengetopt-2.23.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A tool for generating scanners"
+DESCRIPTION="Gengetopt is a tool to write command \
+line option parsing code for C programs."
+HOMEPAGE="https://www.gnu.org/software/gengetopt/"
+COPYRIGHT="1999-2007 Free Software Foundation, Inc."
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/gengetopt/gengetopt-$portVersion.tar.xz"
+CHECKSUM_SHA256="b941aec9011864978dd7fdeb052b1943535824169d2aa2b0e7eae9ab807584ac"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	gengetopt$secondaryArchSuffix = $portVersion
+	cmd:gengetopt$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize
+	cmd:make
+	"
+
+BUILD()
+{
+	autoreconf -fi
```

Comment:
I like to see what it does so usually I work with "autoreconf -vfi", but this works also